photoprism/umbrel-app.yml

```diff
@@ -42,22 +42,7 @@ defaultUsername: "admin"
 deterministicPassword: true
 torOnly: false
 releaseNotes: >-
-  This update brings a ton of new features, performance improvements, and bug fixes, including:
-
-  - new high-resolution vector world map
-  
-  - improved performance, security, and file type support
-
-  - improved translations
-
-  - improved video trascoding
-
-  - improved user interface
-
-  - new themes
-
-  - and more!
-
+  Our latest release includes a redesigned Places view, with the search box moved to the top and a preview for selected clusters at the bottom. We've also added support for Samsung/Google Motion Photos, so you can view them like Apple Live Photos after re-indexing your library. Beyond those highlights, you'll get many usability improvements, new search filters, and fixes for recently discovered issues. A big thank you to everyone who contributed!
 
```

Comment:
I'll also add one more blank line here so it moves to the next line for better visibility.

Reply:
I added one more blank line between the release description and the full release notes footer, if this is what you mean. Thank you for checking the image digest 🙏🏻

Reply:
No worries, Thanks for the contribution !!